jazkarta/shop/browser/checkout.py

```diff
@@ -1,4 +1,10 @@
+import hmac
+import time
+import random
+from authorizenet import apicontractsv1
+from authorizenet.apicontrollers import *
```

Comment:
looks like it wasn't even needed (left over from when I was implementing accepted hosted solution, we now switched to SIM)